src/GridFieldConfigurablePaginator.php

```diff
@@ -64,10 +67,21 @@ public function __construct($itemsPerPage = null, $pageSizes = null)
      * Get the total number of records in the list
      *
      * @return int
+     * @deprecated 4.1.1 Use getTotalItems() instead.
      */
     public function getTotalRecords()
     {
-        return (int) $this->getGridField()->getList()->count();
+        Deprecation::notice('4.1.1', 'Use getTotalItems() instead.');
+        return $this->getTotalItems();
+    }
```

Comment:
Deprecating in favour of the method in the parent class, since the purpose is the same.